packages/components/src/Calendar/YearListBox.tsx

```diff
@@ -54,25 +56,27 @@ const YearListBox = ({ setSelectedDropdown }: YearDropdownProps) => {
       className="grid h-full max-h-[300px] min-w-[300px] grid-cols-3 gap-y-10 overflow-y-scroll p-2"
     >
       {years.map((year, index) => {
-        const isActive = +year.formatted === state.focusedDate.year;
+        const isSelected = +year.formatted === state.focusedDate.year;
 
         return (
           <li className="flex justify-center" key={index}>
             <div
-              ref={isActive ? activeButtonRef : (null as any)}
+              ref={isSelected ? activeButtonRef : (null as any)}
               style={{ height: '100%', width: '100%' }}
             >
               <Button
                 slot="previous"
-                disabled={state.isDisabled}
-                variant={isActive ? 'secondary' : 'text'}
-                size="small"
+                className={cn(
+                  classNames.dateSelectorOption,
```

Comment:
I think a better name would be `calendarListboxButton` or something like this? I think of date if i read `date` but thats just a part of it? What do you think